screen-editor.tex

```diff
@@ -97,6 +97,8 @@ \section{Editor Functionality}
 
 Long press on \widekey{RESTORE} to go into the Freeze Menu.  Then press \megakey{J} to switch joystick ports without having to physically swap the joystick to the other port.
 
-Go to \textbf{Fast mode} with \screentext{POKE 0,65} or use the Freeze Menu.
+Go to \textbf{Fast mode} (40.5MHz) by either typing \screentext{FAST}, or \screentext{POKE 0,65} or use the Freeze Menu.
```

Comment:
Also, the extras section looks a little blunt/informal. Maybe it needs a small introduction?
```
\textbf{Extras}
Below are some extra, useful things you can do whilst in the screen editor:

  * To enter the Freeze menu, long press on \widekey{RESTORE}, then press \megakey{J} to switch joystick ports without 
    having to physically swap the joystick to the other port. You can read more about the Freeze menu, and what else it can do
    on page \pageref{add-label-to-freezer}

  * To enter \textbf{Fast mode} (40.5MHz), either type \screentext{FAST}, \screentext{POKE 0,65}, or use the Freeze Menu.```